server/APICaller.js

```diff
@@ -134,23 +131,23 @@ const reviews = {
 const questions = {
   questionsGET: (req, res) => {
     axios.get(
-      'https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/qa/questions',
+      `${server}/qa/questions`,
```

Comment:
@RookAteMySSD - fyi, going to update these hardcoded calls to use the server string variable